libdevcrypto/gm/sm2/sm2.h

```diff
@@ -21,27 +21,28 @@
  * @date: 2018
  */
 #pragma once
-#include <iostream>
-#include <string>
-using namespace std;
 #include <openssl/sm2.h>
 #include <openssl/sm3.h>
+#include <iostream>
+#include <string>
+#define CRYPTO_LOG(LEVEL) LOG(LEVEL) << "[#CRYPTO] "
```

Comment:
put the ERROR_OUTPUT into the definition of CRYPTO_LOG if you need to output error msg to std::out.

But I don't think you need

Reply:
It's too ugly for logging twice in code

Reply:
I just modified it according to the size of everyone, maybe we need to modify our standards?

Reply:
like line 56~59 in SecureInitializer.cpp

Reply:
modify it now,maybe this can be better ; )